doc/install.rst

```diff
@@ -0,0 +1,65 @@
+:orphan:
+
+Installation
+============
+
+Dependencies
+------------
+
+* ``mne`` (>=1.0)
+* ``numpy`` (>=1.14)
+* ``scipy`` (>=1.5.0)
+* ``joblib`` (>=1.0.0)
+* ``scikit-learn`` (>=1.1)
+* ``torch`` (for running pytorch neural networks)
+* ``python-picard`` (for running ICA)
+* ``matplotlib`` (optional, for using the interactive data inspector)
```

Comment:
Minor detail for now, but this should be updated when #16 is resolved. I'm guessing matplotlib and joblib are not required?